CodingClubSource/CodingClub/StringLibrary.cpp

```diff
@@ -16,12 +16,17 @@ std::string removeVowels(const std::string input)
 		case 'o':
 		case 'u':
 			break;
```

Comment:
Doesn't need this break statement, all of them can fall down into the one on line 24